packages/vaadin-virtual-list/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
Let's add a top-level yarn script to run this demo, so we don't need to run tests in debug mode for it.

Reply:
The file would get overridden by the virtual-list implementation PR, so removed it from this PR